pkg/cri/sbserver/streaming.go

```diff
@@ -158,7 +160,10 @@ func (s *streamRuntime) PortForward(podSandboxID string, port int32, stream io.R
 		return fmt.Errorf("invalid port %d", port)
 	}
 	ctx := ctrdutil.NamespacedContext()
-	return s.c.portForward(ctx, podSandboxID, port, stream)
+	if pf, ok := s.c.sandboxController.(sandbox.PortForward); ok {
```

Comment:
I think this needs to be updated.
We `c.getSandboxController` to obtain controller interface.
Also shim controllers go throught containerd backend.